handlers/create.go

```diff
@@ -31,6 +31,11 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 		return handleByIdError(err, event, apiClient)
 	}
 
+	// If name is not supplied, error out
```

Comment:
We should handle this in the API by adding something like this:
```
    	"name": {
    		"type": "string",
            "nullable": false,
            "required": true
    	},
```
to machine.json (where the other fields are defined)